scripts/Tools/case.build

```diff
@@ -54,11 +54,31 @@ OR
     parser.add_argument("-m", "--model-only", action="store_true",
                         help="Assume shared libs already built")
 
+    comps = ("atm", "ice", "glc", "lnd", "ocn", "rof", "wav", "cpl")
+    libs  = ("shr", "mct", "pio", "tim")
+    allobjs = comps + libs
+
+    parser.add_argument("--clean", nargs="*", choices=allobjs,
+                        help="objects to clean"
+                        "if no arguments then clean only atm, ice, glc, lnd, ocn, rof, wav objects")
+
+    parser.add_argument("--cleanall", action="store_true",
+                        help="clean all objects ")
+
     args = parser.parse_args(args[1:])
 
+    cleanlist = []
```

Comment:
could be slightly cleaner with a ternary:

    cleanlist = allobjs if args.cleanall else p[